apps/file/models.py

```
@@ -19,3 +19,11 @@ class Type(models.TextChoices):
     file = models.FileField(
         verbose_name=_("File"), max_length=255, upload_to="imported_files/"
     )
+    file_size = models.PositiveIntegerField(
+        verbose_name=_("File size"),
+    )
+
+    def save(self, *args, **kwargs):
+        self.full_clean()  # To enforce model CharField validation for chocies.
```

Let's raise an error message here.

---

It automatically raises an error.